### PR TITLE
Nawqaan 478

### PR DESCRIPTION
**Pull request by cschroed-usgs**

### CODE REVIEW

File: nar-webservices/src/test/java/gov/usgs/cida/nar/transform/ExcelLeadingZeroIdFixTransformTest.java [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+/**
+ *
+ * @author dmsibley
+ */
+public class ExcelLeadingZeroIdFixTransformTest {
+
+    private static final Logger log = LoggerFactory.getLogger(WaterYearTransform.class);
+    public static final Column QW_ID_COLUMN = new SimpleColumn(SosAggregationService.SITE_QW_ID_IN_COL);
+    public static final Column FLOW_ID_COLUMN = new SimpleColumn(SosAggregationService.SITE_FLOW_ID_IN_COL);
```
Comment (cschroed-usgs) [inline nar-webservices/src/test/java/gov/usgs/cida/nar/transform/ExcelLeadingZeroIdFixTransformTest.java]: This was the reason that I put all of those SosAggregationService strings public. Now I'm not sure I needed to. Should I change them back, and just use any old column name I please in the test?

File: nar-webservices/src/main/java/gov/usgs/cida/nar/service/plan/FixLeadingZeroIdInExcelStepper.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+import gov.usgs.cida.nar.transform.ExcelLeadingZeroIdFixTransform;
+import gov.usgs.cida.nude.column.ColumnGrouping;
+import gov.usgs.cida.nude.column.SimpleColumn;
+import gov.usgs.cida.nude.filter.FilterStageBuilder;
+import gov.usgs.cida.nude.filter.FilterStep;
+import gov.usgs.cida.nude.filter.NudeFilterBuilder;
+import gov.usgs.cida.nude.plan.PlanStep;
+import java.util.List;
+
+public class FixLeadingZeroIdInExcelStepper {
```
Comment (cschroed-usgs) [inline nar-webservices/src/main/java/gov/usgs/cida/nar/service/plan/FixLeadingZeroIdInExcelStepper.java]: This name is weird. Should I change it? What to?

Comment (jiwalker-usgs) [inline nar-webservices/src/main/java/gov/usgs/cida/nar/service/plan/FixLeadingZeroIdInExcelStepper.java]: StepBuilder?

File: nar-webservices/src/test/java/gov/usgs/cida/nar/transform/ExcelLeadingZeroIdFixTransformTest.java (hunk 1 above)
Comment (jiwalker-usgs) [inline nar-webservices/src/test/java/gov/usgs/cida/nar/transform/ExcelLeadingZeroIdFixTransformTest.java]: I think you're right to make them public for reuse.  It ends up being a bit ugly, but not the worst thing in the world.

**Comment by jiwalker-usgs** [comment]
:shipit:

**Comment by cschroed-usgs** [comment]
Thanks JW.

**Pull request merged**